src/rard/research/models/base.py

```diff
@@ -571,18 +571,39 @@ def mentioned_in_list(self):
         related_name="%(class)s_duplicate_anonfragments",
     )
 
+    # if converted to a testimonium, we want to transfer duplicates
+    duplicate_ts = models.ManyToManyField(
+        "Testimonium",
+        blank=True,
+        related_name="%(class)s_duplicate_ts",
+    )
+
     @property
     def duplicates_list(self):
         duplicates = list(self.duplicate_frags.all())
```

Comment:
I've simplified this in the latest commit: as long as we make sure duplicate relationships are always bi-directional, we shouldn't have to look at the reverse relationships.